svgdigitizer/__main__.py

```diff
@@ -128,8 +128,12 @@ def plot(svg, sampling_interval):
     default=None,
     help="write output files to this directory",
 )
+@click.option('--skewed',
+              is_flag=True
+              help="If the plot axes are skewed use '--skewed' to align the axes along the markers instead of orthogonally (default)."
```

Comment:
```suggestion
              help="Detect non-orthogonal skewed axes going through the markers instead of assuming that axes are perfectly horizontal and vertical."
```